.gitignore

```diff
@@ -101,7 +101,7 @@ solutions/swb-reference/integration-tests/config/*.yaml
 solutions/swb-reference/src/config/*.json
 solutions/swb-reference/.aws-sam/
 
-
+out/
```

Comment:
why was this added?

Reply:
when I build the world, a folder is generated somewhere in SWB land called out/ which was polluting my changed files with about 40 unneeded files.

Reply:
What directory was this generated in? I'm surprised it wasnt included in `.gitignore` already

Reply:
That was recent change related to moving ui stuff. L114 has correct path. Just delete everything from that folder and it won't show up again

Reply:
removed